internal/pipe/brew/template.go

```diff
@@ -14,9 +14,11 @@ type templateData struct {
 	Dependencies     []string
 	Conflicts        []string
 	Tests            []string
+	CustomRequire    string
 }
 
-const formulaTemplate = `class {{ .Name }} < Formula
+const formulaTemplate = `{{- if .CustomRequire }}require_relative "{{ .CustomRequire }}"{{- else}}# No require_relative{{- end }}
```

Comment:
I think we can drop the `# no require_relative` comment...

Reply:
I was hoping you wouldn't say that :smile_cat: it will leave an empty line at the top of the file unless you know a good way to trim it?

Reply:
I'll play with this today and fix - think I've found how